dali/dfu/dfuwu.cpp

```diff
@@ -64,9 +75,9 @@ static void removeTree(IPropertyTree *root,const char *name)
         root->removeTree(t);
 }
 
-static StringBuffer &newWUID(StringBuffer &wuid)
+static StringBuffer &newWUID(StringBuffer &wuid, const char prefix = 'D')
```

Comment:
minor: prob. better to avoid defaults like this, can lead to confusion, better for the 1 pre-existing call in createWorkUnit() to explicitly pass 'D'

Reply:
Agreed, will change.